Article/Article.js

```diff
@@ -2,28 +2,34 @@
 
 class Article {
   constructor(domElement) {
-    // assign this.domElement to the passed in domElement
-    this.domElement;
-    // create a reference to the ".expandButton" class. 
-    this.expandButton;
-    // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
-    // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    // assign this.domElement to the passed in domElement [DONE]
+    this.domElement = domElement;
+    // create a reference to the ".expandButton" class. [DONE]
+    this.expandButton = domElement.querySelector('.expandButton');
+    console.log('Creating Expand Button reference', domElement, this.expandButton);
+    // Using your expandButton reference, update the text on your expandButton to say "expand" [DONE]
+    this.expandButton.textContent = 'Expand';
+    // Set a click handler on the expandButton reference, calling the expandArticle method. [DONE]
+    this.expandButton.addEventListener('click', () => this.expandArticle())
   }
 
   expandArticle() {
-    // Using our reference to the domElement, toggle a class to expand or hide the article.
+    // Using our reference to the domElement, toggle a class to expand or hide the article. [DONE]
+    this.domElement.classList.toggle('article-open')
```

Comment:
One thing I would add here is a ternary operator, that would let the user know if the article is opened or closed, it just gives a little bit of feedback to the user and can make things a bit more intuitive.
```
 this.domElement.classList.contains('article-open') ? 
this.expandButton.textContent = 'close' : this.expandButton.textContent = 'expand';
```